ext/lib/mgmt/mcumgr/samples/smp_svr/zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# Top-level CMakeLists.txt for the skeleton application.
```

Comment:
Samples must have a README.rst file in their directory (Next to prj.conf, CMakeLists.txt, etc.).

Perhaps some of the information in samples/smp_ver/README.md could be moved into this file.

Reply:
OK, good to know.  When I move the sample app into the top-level `samples` directory (which I will do soon), I will include a `README.rst` file.